stdlib/3/builtins.pyi

```diff
@@ -472,7 +472,7 @@ class slice:
     start = 0
     step = 0
     stop = 0
-    def __init__(self, start: int, stop: int = 0, step: int = 0) -> None: ...
+    def __init__(self, start: Optional[int], stop: Optional[int] = 0, step: Optional[int] = 0) -> None: ...
```

Comment:
Please change the `step`'s default value to be `None`. In fact, it *cannot* be 0 :) Same for `stop`, the default value is `None`, behavior is different if 0 is given.